java/athena/src/test/java/sleeper/athena/FilterTranslatorTest.java

```diff
@@ -90,7 +90,7 @@ public void shouldTranslateEmptyMapIntoNull() {
     @Test
     public void shouldCreateIntegerRangePredicateFromSortedSetContainingRange() throws IOException {
         // Given
-        String dataFile = new File(tempDir.newFolder(), "test.parquet").getAbsolutePath();
+        String dataFile = new File(createTempDirectory(tempDir.toPath(), null).toString(), "test.parquet").getAbsolutePath();
```

Comment:
This is not important as what it's doing is consistent with how it's used elsewhere. I do think it would be nice to have a helper method that stops you needing to repeat the code to get a `String dataFile` variable.